deployment/aks-periscope.yaml

```diff
@@ -76,6 +76,8 @@ spec:
         envFrom:
         - configMapRef:
             name: containerlogs-config
+        - configMapRef:
```

Comment:
💡 Just a note and observation to keep in mind: 

* These yaml changes for master will affect few consuming tools, like `vscode` ecosystem and `az-cli`, so I will emphasise to keep this in mind for design/ logic flow.
* Just as thoughts: If this gets any complicated, I would recommend exploring separate `yaml` idea.

Thanks guys.

Reply:
Thanks Tatsat, In general we are trying to keep the yaml generic.

Instead of having a copy of yaml, does it makes sense to have a helm chart? CLUSTER_TYPE=appliance is other scenairo that might be interested in periscope (appliance is another type of AKS cluster). When this comes in we would end up with one more yaml file.

I have given a first short at helm chart here

[https://github.com/safeermohammed/aks-periscope/tree/master/charts/azure-k8s-periscope](https://github.com/safeermohammed/aks-periscope/tree/master/charts/azure-k8s-periscope)

This if for reference, let me know if this makes sense?

Reply:
Also let us know if vscode breaks when this RP is merged, we can create a copy of arc specific yaml file for now.

Reply:
Ahhhh! Yes thanks @safeermohammed - helm chart is just what we need here. 

Will pull this into my PR and adapt planned changes to leverage it. We can deprecate the existing aks-periscope.yaml and leave as-is until cli + vscode can move to consume the helm chart instead.

Reply:
💡 Note: We need to make sure that the impact to the external tool, at-least to `az-cli` and `vscode` eco-system is **non-breaking** in case it **if we** need a discuss why will need those breaking changes at all and what are the alternatives.

Reply:
@Tatsinnit @safeermohammed with a single deployment yaml, which currently is pulled on demand by vscode and CLI (both of which are fixed to target master HEAD), how would you suggest we go about merging breaking changes to the deployment yaml - such as https://github.com/Azure/aks-periscope/pull/52